navigation/src/main/kotlin/com/imashnake/animite/navigation/TopLevelDestinations.kt

```diff
@@ -0,0 +1,20 @@
+package com.imashnake.animite.navigation
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ProfileRoute(
```

Comment:
Not sure if I'm a fan of this, but it suddenly becomes a bigger refactor to not have it here